starter-code/apple-pie-starter-code/index.html

```diff
@@ -1,35 +1,65 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <title>Apple Pie Recipe</title>
-</head>
-<body>
-Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
-quite like it. It will always be my favorite and has won me several first place
-prizes in local competitions. I hope it becomes one of your favorites as well!
+  <head>
+    <link rel="stylesheet" href="css/main.css" />
+    <title>Apple Pie Recipe</title>
+  </head>
+  <body>
+    <header class="header">
+      <h1>
+        Apple Pies
+      </h1>
+    </header>
+  
+    <section id="intro">
+        <p>
+            This was my grandmother's apple pie recipe. I have never seen another
+            one quite like it. It will always be my favorite and has won me several
+            first place prizes in local competitions. I hope it becomes one of your
+            favorites as well!
+          </p>
+          <img src="images/cooking-info.png" alt="okay" />
```

Comment:
It's great that you've used alt tags, but you should give them meaningful text eg "cooking info"